File: src/main/java/com/GoldenMine/parser/postposition/IPostPosition부사어에서.java
package com.GoldenMine.parser.postposition;

public class IPostPosition부사어에서 implements IPostPosition {
    @Override
    public String getType() {
        return "부사어에서";
    }

    @Override
    public int verify(CharSequence last) {
        if(last.length() >= 2) {

            char ch = last.charAt(last.length() - 2);
            char ch2 = last.charAt(last.length() - 1);

            return (ch == '에' && ch2 =='서') ? 2 : -1;
        } else {
            return -1;
        }
    }

    @Override
    public JosaCommunity getJosaCommunity() {
        return JosaCommunity.격조사;
    }
}

File: src/main/java/kr/goldenmine/parser/postposition/IPostPosition부사어에서.kt
package kr.goldenmine.parser.postposition

class IPostPosition부사어에서 : IPostPosition {
    override val type: String
        get() = "부사어에서"

    override val josaCommunity: JosaCommunity
        get() = JosaCommunity.격조사

    override fun verify(last: CharSequence): Int {
        if (last.length >= 2) {

            val ch = last[last.length - 2]
            val ch2 = last[last.length - 1]

            return if (ch == '에' && ch2 == '서') 2 else -1
        } else {
            return -1
        }
    }
}
